editor/gui/scene_tree_editor.cpp

```diff
@@ -968,15 +968,34 @@ void SceneTreeEditor::_rename_node(Node *p_node, const String &p_name) {
 	String new_name = p_name.validate_node_name();
 
 	if (new_name != p_name) {
-		error->set_text(TTR("Invalid node name, the following characters are not allowed:") + "\n" + String::get_invalid_node_name_characters());
-		error->popup_centered();
+		String text = TTR("Invalid node name, the following characters are not allowed:") + "\n" + String::get_invalid_node_name_characters();
+		if (error->is_visible()) {
```

Comment:
Not sure if this condition is necessary; it's not really something that can happen here.

Reply:
Unfortunately this can happen with batch rename if one node is erroring because of unique name already in use and another node because of invalid character later.

Reply:
> I'll try to show an actual case of it happening tomorrow.

Right...